source/PluginDev/Assets/GooglePlayGames/BasicApi/RequestPermissionStatusCodes.cs

```diff
@@ -0,0 +1,35 @@
+namespace GooglePlayGames.BasicApi
+{
+    public enum RequestPermissionStatusCodes
```

Comment:
To be consistent le'ts rename it to RequestPermissionUIStatus

Reply:
done